internal/backend/remote-state/azure/backend.go

```diff
@@ -49,6 +49,13 @@ func New() backend.Backend {
 				Description: "The blob key to use in the Storage Container.",
 			},
 
+			"lookup_blob_endpoint": {
```

Comment:
Could we add a test for this? And we should update the documentation to include this as well

Reply:
Cc @jaredfholgate